src/test/kotlin/org/rust/lang/core/resolve/RsCfgAttrResolveTest.kt

```diff
@@ -752,4 +752,30 @@ class RsCfgAttrResolveTest : RsResolveTestBase() {
             func();
         } //^ unresolved
      """)
+
+    fun `test unknown cfg options are disabled`() = checkByCode("""
+        #[cfg(unknown_cfg_option)]
+        fn foo() {}
+        #[cfg(not(unknown_cfg_option))]
+        fn foo() {}
+         //X
+
+        fn main() {
+            foo();
+          //^
+        }
+    """)
+
+    fun `test unknown cargo features are disabled`() = checkByCode("""
+        #[cfg(feature = "unknown_cargo_feature")]
+        fn foo() {}
+        #[cfg(not(feature = "unknown_cargo_feature"))]
+        fn foo() {}
+         //X
+
+        fn main() {
+            foo();
+          //^
+        }
+    """)
```

Comment:
Let's add more tests of using unknown options and features in complex predicates.
Also, some tests for stdlib features (like `bootstrap` and `test`) would be very helpful as well. However, I'm not sure if such tests are easy to implement, what do you think?

Reply:
Oh, I don't know how to check evaluation in the stdlib...